apps/storybook/src/TiledHeatmapMesh.stories.tsx

```diff
@@ -1,390 +0,0 @@
-import type {
```

Comment:
This file moves into a new `TiledHeatmapMesh` folder. The API classes, utilities and the `TileParams` model all move to separate files.